digitalpalireader/content/js/dpr_pal.js

```diff
@@ -117,8 +124,9 @@ console.log('Loading DPR_PAL...');
       $(bottomFrameSelector).slideToggle();
       if ($(bottomFrameSelector).is(":visible")) {
         $(".rotate").toggleClass("down");
-        $(".rotate").hasClass("down")?$("#paliTextContent").addClass("COLLAPSE"):$("#paliTextContent").removeClass("COLLAPSE");
+        $(".rotate").hasClass("down") ? bottomFrameUp=true : bottomFrameUp=false;
```

Comment:
It can be simplified to

```
bottomFrameUp = $(".rotate").hasClass("down");
```

Reply:
Even better I am wondering if instead of a variable bottomFrameUp, we can just have a function

```
BottomFrameUp = () => $(".rotate").hasClass("down");
```

Reply:
Good comment, I was also not happy with the variable. Changed it.